### PR TITLE
[WEB-611] Add message sent message

### PR DESCRIPTION
**Pull request by tomsonpl**
screen shot 2017-02-10 at 16 43 37

### CODE REVIEW

File: static/parse-hosting/index.html [hunk 1]
```diff
@@ -159,11 +169,19 @@
...
         const submitRef = '&submit=Submit';
         const submitURL = `${baseURL}${question}=${emaildata}${submitRef}`;
         $(this)[0].action=submitURL;
-        $('#email-form').val('Thank you!');
+        $('input').click(function(){
+            $('.jsFormMessage p').html('');
+        });
+        $('#email-form').val('');
+        $('.jsFormMessage p').html(messageSent);
+        $('.jsFormMessage').addClass('form__message--success').val('').show();
```
Comment (wpasinski) [inline static/parse-hosting/index.html]: `.val('')`?

File: static/parse-hosting/index.html [hunk 2]
```diff
@@ -95,34 +97,39 @@
...
       }
-
-      .form input,
-      .form__message {
-        margin-bottom: 20px;
-      }
     }
     
     @media (max-width: 768px) {
+
```
Comment (wpasinski) [inline static/parse-hosting/index.html]: please fix all white spaces.

File: static/parse-hosting/index.html [hunk 3]
```diff
@@ -95,34 +97,39 @@
...
         box-shadow: 0 0 50px rgba(0, 0, 0, .37);
       }
-
-      .form input,
-      .form__message {
-        margin-bottom: 20px;
-      }
     }
     
     @media (max-width: 768px) {
```
Comment (wpasinski) [inline static/parse-hosting/index.html]: please keep it mobile first. and why changing RWD behaviour which is already coded from 540px?

File: static/parse-hosting/index.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
     * { box-sizing: border-box; }
 
     body {
-      background: #fff;
+      background: #1976d2;
```
Comment (wpasinski) [inline static/parse-hosting/index.html]: switch background color in a media query.

File: static/parse-hosting/index.html (hunk 1 above)
Comment (tomsonpl) [inline static/parse-hosting/index.html]: it clears the input.  Do you think we shouldn't clear the input after sending?

**Pull request merged**